### PR TITLE
Feat/setup

### PR DESCRIPTION
**Pull request by whawari**

### CODE REVIEW

**Review by thelebdev — CHANGES_REQUESTED** [review]
`.env.example` is the only file that should be committed.

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        ""
+      )}
+      <div
+        id="router-components"
+        style={{ margin: isSidebar ? "" : "var(--header-height) 0 0" }}
+      >
+        <Header toggleSidebar={toggleSidebar} isSidebar={isSidebar} />
+        <Switch>
+          <Route exact path="/" component={Homepage} />
+          <Route exact path="/add-listing" component={AddListing} />
```
Comment (thelebdev) [inline src/components/App.js]: I suggest using the naming convention we agreed on earlier. `/object/action/child-object/action`

File: src/components/listings/AddListing.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import "./index.css";
+import "react-day-picker/lib/style.css";
+
+const AddListing = () => {
+  const handleInputChange = (event) => {
+    const target = event.target;
+    const name = target.name;
+    // const value = target.value;
+
+    switch (name) {
```
Comment (thelebdev) [inline src/components/listings/AddListing.js:15]: Why are the cases empty?

**Review by thelebdev — CHANGES_REQUESTED** [review]

File: src/components/App.js (hunk 1 above)
Comment (whawari) [reply src/components/App.js]: Already did it

File: src/components/listings/AddListing.js (hunk 2 above)
Comment (whawari) [reply src/components/listings/AddListing.js:15]: I building the cases logic, wait for the next pull request.

**Review by thelebdev — APPROVED** [review]

**Pull request merged**